### PR TITLE
Completion -> client

### LINKED ISSUE #386: openai.ChatCompletion.create() -> client.chat.completions.create()
```[tasklist]
### Tasks
- [x] create OpenAI client
- [x] use client instead of Completion
```

### PR DESCRIPTION
**Pull request by sonichi**
## Why are these changes needed?

openai package change

use client instead of Completion

## Related issue number

#203 
close #386 #387

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Review by Steellgold — APPROVED** [review]

**Review by alhridoy — APPROVED** [review]

**Review by rabbabansh — APPROVED** [review]

**Review by Gourav2609 — APPROVED** [review]

**Review by AllenJShi — APPROVED** [review]
LGTM

**Review by JieyuZ2 — APPROVED** [review]

**Review by ragyabraham — APPROVED** [review]
LGTM

**Review by rickyloynd-microsoft — APPROVED** [review]

**Review by rabbabansh — APPROVED** [review]

File: autogen/code_utils.py
Comment (pcdeadeasy) [inline autogen/code_utils.py:1]: Remove the commented block of code

**Review by rabbabansh — APPROVED** [review]

**Review by pcdeadeasy — COMMENTED** [review]
Looks good except I see a bunch of Deprecated functions, does it means they are still available to be used?

**Comment by sonichi** [comment]
> Looks good except I see a bunch of Deprecated functions, does it means they are still available to be used?

They are available for openai<1

**Comment by pcdeadeasy** [comment]
> > Looks good except I see a bunch of Deprecated functions, does it means they are still available to be used?
> 
> They are available for openai<1

Can we say that then, instead of just saying Deprecated.

**Comment by sonichi** [comment]
> > > Looks good except I see a bunch of Deprecated functions, does it means they are still available to be used?
> > 
> > 
> > They are available for openai<1
> 
> Can we say that then, instead of just saying Deprecated.

Sure. I'll leave it to other devs.

**Review by manish7017 — COMMENTED** [review]
lgtm.

**Pull request merged**